<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="24827" started-at="2016-10-14T06:29:51Z" finished-at="2016-10-14T06:30:16Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="5106" started-at="2016-10-14T06:29:51Z" finished-at="2016-10-14T06:29:57Z">
      <class name="AutomationTests.Login">
        <test-method status="PASS" signature="callLogin()[pri:0, instance:AutomationTests.Login@1b1426f4]" name="callLogin" duration-ms="4779" started-at="2016-10-14T11:59:51Z" finished-at="2016-10-14T11:59:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- callLogin -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:AutomationTests.Login@1b1426f4]" name="tearDown" is-config="true" duration-ms="307" started-at="2016-10-14T11:59:56Z" finished-at="2016-10-14T11:59:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=callLogin status=SUCCESS method=Login.callLogin()[pri:0, instance:AutomationTests.Login@1b1426f4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AutomationTests.Login -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="3831" started-at="2016-10-14T06:29:57Z" finished-at="2016-10-14T06:30:00Z">
      <class name="AutomationTests.CreateProject">
        <test-method status="FAIL" signature="testmethod()[pri:0, instance:AutomationTests.CreateProject@32ee6fee]" name="testmethod" duration-ms="3427" started-at="2016-10-14T11:59:57Z" finished-at="2016-10-14T12:00:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at utilities.functionLibrary.createProject(functionLibrary.java:110)
	at AutomationTests.CreateProject.testmethod(CreateProject.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:AutomationTests.CreateProject@32ee6fee]" name="tearDown" is-config="true" duration-ms="400" started-at="2016-10-14T12:00:00Z" finished-at="2016-10-14T12:00:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testmethod status=FAILURE method=CreateProject.testmethod()[pri:0, instance:AutomationTests.CreateProject@32ee6fee] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AutomationTests.CreateProject -->
    </test> <!-- Test2 -->
    <test name="Test3" duration-ms="3376" started-at="2016-10-14T06:30:00Z" finished-at="2016-10-14T06:30:04Z">
      <class name="AutomationTests.CreateProject_negative">
        <test-method status="PASS" signature="testMethod()[pri:0, instance:AutomationTests.CreateProject_negative@6da21078]" name="testMethod" duration-ms="3284" started-at="2016-10-14T12:00:00Z" finished-at="2016-10-14T12:00:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:AutomationTests.CreateProject_negative@6da21078]" name="tearDown" is-config="true" duration-ms="89" started-at="2016-10-14T12:00:04Z" finished-at="2016-10-14T12:00:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testMethod status=SUCCESS method=CreateProject_negative.testMethod()[pri:0, instance:AutomationTests.CreateProject_negative@6da21078] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- AutomationTests.CreateProject_negative -->
    </test> <!-- Test3 -->
    <test name="Test4" duration-ms="12493" started-at="2016-10-14T06:30:04Z" finished-at="2016-10-14T06:30:16Z">
      <class name="AutomationTests.Createcrawler">
        <test-method status="PASS" signature="testmethod()[pri:0, instance:AutomationTests.Createcrawler@35aea049]" name="testmethod" duration-ms="12491" started-at="2016-10-14T12:00:04Z" finished-at="2016-10-14T12:00:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testmethod -->
      </class> <!-- AutomationTests.Createcrawler -->
    </test> <!-- Test4 -->
  </suite> <!-- Suite -->
</testng-results>
